include/openssl/opensslv.h

```diff
@@ -40,7 +40,9 @@ extern "C" {
  *  major minor fix final patch/beta)
  */
 # define OPENSSL_VERSION_NUMBER  0x10102000L
-# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.2-dev  xx XXX xxxx"
+# define OPENSSL_VERSION_NUMBER_TEXT "1.1.2-dev"
+# define OPENSSL_VERSION_TEXT "OpenSSL " \
+                              OPENSSL_VERSION_NUMBER_TEXT "  xx XXX xxxx"
```

Comment:
Hmmm, I was thinking like this:

``` C
# define OPENSSL_VERSION_TEXT \
        "OpenSSL " OPENSSL_VERSION_NUMBER_TEXT "  xx XXX xxxx"
```

(the whole string composition itself on one line, see?)

Reply:
Ah sorry, I've updated this now.